plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -589,15 +589,10 @@ global proc mayaUsdMenu_pushBackToUSD(string $obj)
         }
     }
     if (size($obj)) {
-        python("from mayaUsd.lib import PrimUpdaterManager; PrimUpdaterManager.mergeToUsd('" + $obj + "');");
```

Comment:
In MEL, many places where there used to be Python code calling into C++ are replaced by proper commands.